js/package.json

```diff
@@ -80,11 +80,15 @@
     "openai": "^4.50.0",
     "pusher-js": "8.4.0-rc2",
     "resolve-package-path": "^4.0.3",
+    "rollup-plugin-terser": "^7.0.2",
     "ts-node": "^10.9.2",
     "uuid": "^10.0.0",
     "winston": "^3.13.1",
     "zod": "^3.23.8",
     "zod-to-json-schema": "^3.23.2"
+  },
+  "dependencies": {
```

Comment:
Why are dependencies all emptied out

Reply:
Bundling all the stuff. No dependencies needed as of now. 

For others like FE SDK, we will have peerDependencies.